src/client/app/components/about.ts

```diff
@@ -50,28 +63,75 @@ export class About implements AfterViewInit {
     currentChosenLanguage = '';
     originalLanguage = '';
 
+    configs = [];
+    selectedConfig = '';
+
+    constructor(
+        private _effects: UIEffects,
+        private _changeDetector: ChangeDetectorRef
+    ) {}
+
     ngAfterViewInit() {
         this.availableLanguages = getAvailableLanguages();
         this.currentChosenLanguage = getDisplayLanguage();
         this.originalLanguage = this.currentChosenLanguage;
+        this.configs.push(
+            { name: 'Production', value: 'production' },
+            { name: 'Beta', value: 'insiders' },
+            { name: 'Alpha', value: 'edge' },
+        );
+
+        // User can only navigate to localhost if they've sideloaded local manifest
+        let showLocalConfig = environment.current.config.name === config['local'].name || (window.localStorage.getItem(PLAYGROUND_ORIGIN) && window.localStorage.getItem(PLAYGROUND_ORIGIN).indexOf('localhost') !== -1);
```

Comment:
I actually removed one of the conditions, so one line should be enough now? Can fix if still necessary.